lib/mixins.js

```diff
@@ -41,10 +41,12 @@ MixinProvider.prototype.applyMixin = function applyMixin(modelClass, name, optio
     // Try model name
     var model = this.modelBuilder.getModel(name);
     if (model) {
-      debug('Mixin is resolved to a model: %s', name);
+      debug('Mixin "' + name + '" resolves to model: %s', modelClass.modelName);
```

Comment:
This is not correct. Mixin "name" is resolved to a model called "name" in this branch, and then it is applied to another model called "modelClass.modelName".

It's better to avoid making unrelated code changes, as it makes code archaeology more difficult.

Please revert this line.